src/modules/collection/utils.ts

```diff
@@ -38,7 +38,10 @@ export function getSaleAddress(chainId: ChainId) {
 }
 
 export function getCollectionEditorURL(collection: Collection, items: Item[]): string {
-  return locations.itemEditor({ collectionId: collection.id, itemId: items.length > 0 ? items[0].id : undefined })
+  return locations.itemEditor({
+    collectionId: collection.id,
+    itemId: items.length > 0 ? (items.find(item => item.type === ItemType.WEARABLE) || items[0]).id : undefined
```

Comment:
We already have this logic in a [location selector](https://github.com/decentraland/builder/blob/0708387ff1cf11d781dc6e38d477e9b69845332c/src/modules/location/selectors.ts#L75C6-L75C6) called `getSelectedItemId`.

WDYT about creating a new function `getFirstWearableOrItem` (or something similar), and using it in both places?

Reply:
I like the idea, working on it :)